src/codeu/chat/client/simplegui/ChatGuiFX.java

```diff
@@ -21,19 +20,18 @@
     private final static Logger.Log LOG = Logger.newLog(ChatGuiFX.class);
 
 
-    private static final double WIDTH = 1000;
-    private static final double HEIGHT = 500;
+    private static final double WINDOW_WIDTH = 1000;
+    private static final double WINDOW_HEIGHT = 500;
 
     private Stage thestage;                         // Holds the scene that user is currently viewing
     private Scene signInScene, mainScene;           // Scenes to hold all the elements for each page
-    private BorderPane signInPane;                  // Root panes for both pages (panes provide structure to scene elements)
-    private FlowPane signInLabelPane;               // Panes to hold labels (so that they can be aligned in the root pane)
-    private GridPane inputPane;                     // Pane to position username/password input fields and labels, as well as sign-in button
-    private Label signInLabel;                    // Labels for the main page and sign in page
-    private Button signInButton;                    // Buttons to switch between main page and sign in page
-    private TextField userInput, passInput;         // Text fields for entering username and password
+    private Button signInButton;
+    private TextField userInput;
+    private PasswordField passInput;                // Takes input for username and password
```

Comment:
Some of these comments aren't really necessary. The variable names are already descriptive enough (which is a good thing!), so you don't really need comments that explain them again. Maybe rename `userInput` to `usernameInput` and you can get rid of this comment? Similar for some of the other comments in this class: comments should explain *why* instead of *what*, if that makes sense.